packages/main/src/methods/count-tokens.ts

```diff
@@ -28,7 +28,7 @@ export async function countTokens(
   params: CountTokensRequest,
   requestOptions?: RequestOptions,
 ): Promise<CountTokensResponse> {
-  const url = new RequestUrl(model, Task.COUNT_TOKENS, apiKey, false);
+  const url = new RequestUrl(model, Task.COUNT_TOKENS, apiKey, false, {});
```

Comment:
Shouldn't this be the requestOptions? Otherwise the version won't get put in the url correctly.